activesupport/test/number_helper_test.rb

```diff
@@ -249,7 +253,7 @@ def test_number_number_to_human_size
           assert_equal "1.12 TB",    number_helper.number_to_human_size(1234567890123)
           assert_equal "1.1 PB",   number_helper.number_to_human_size(1234567890123456)
           assert_equal "1.07 EB",   number_helper.number_to_human_size(1234567890123456789)
-          assert_equal "1030 EB",   number_helper.number_to_human_size(exabytes(1026))
```

Comment:
Why this assertion needed to change?

Reply:
@rafaelfranca
Thank you for your review 🙇‍♀️.

I've added zettabyte, so now
number_helper.number_to_human_size(exabytes(1026)) returns "1 ZB".

For the test of exabytes,
I had to update the test to
assert_equal "1020 EB", number_helper.number_to_human_size(exabytes(1023))
(less than 1024).